<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Advisor</class>
 <widget class="QWidget" name="Advisor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Advisor</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>680</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>680</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>680</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 10px 10px 10px 10px;
background-color: rgb(52, 53, 64);
</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>778</width>
      <height>658</height>
     </rect>
    </property>
    <widget class="QGroupBox" name="group_start_screen">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>140</y>
       <width>521</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px;
backgroung: none;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>0</y>
        <width>371</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Welcome to BMCC Virtual Advisor Assistant Bot</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Press start to begin</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_start">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid; 
border-radius:10px;
border-color: rgb(175, 193, 220);
background-color:rgb(58, 59, 59)
</string>
         </property>
         <property name="text">
          <string>START</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>680</y>
     <width>800</width>
     <height>120</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>120</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>120</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(33, 34, 34)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QPushButton" name="btn_send">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid; 
border-radius:10px;
background-color:#00AB66;</string>
    </property>
    <property name="text">
     <string>Send Message</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txt_message">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>611</width>
      <height>81</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid;
border-radius:10px;
background-color:rgb(58, 59, 59);

padding: 10px 10px 10px 10px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_login">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>70</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid; 
border-radius:10px;
background-color:rgb(58, 59, 59)</string>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
